Documents/Automation/EncryptRootVolume/Documents/npark-encryptrootvolume.json

```diff
@@ -158,7 +174,7 @@
       "inputs": {
         "DocumentName": "AWS-StopEC2Instance",
         "RuntimeParameters": {
-          "InstanceId": "{{instanceId}}"
+          "InstanceId": "{{InstanceId}}"
         }
       },
       "name": "StopInstance",
```

Comment:
You are using different naming conventions for the step names. Pick one. I suggest camelCase.

Reply:
I've applied consistent conventions.  Please review.